test/test_testing.py

```diff
@@ -2699,6 +2700,99 @@ def test_container_pebble_ready(self):
         )
 
 
+class TestNetwork(unittest.TestCase):
+    def test_add_network_defaults(self):
```

Comment:
A general comment, we have found that adding a docstring for tests describing what they do is quite helpful with understanding a given test case. It takes a lot of guesswork otherwise to figure out what the test does. Here is an example of how it could be done: https://github.com/jdkandersson/flake8-test-docs

Reply:
In my opinion the test name is enough here. "Test that add_network works with default arguments".